test/e2e/kubernetes/node/node.go

```diff
@@ -61,12 +61,13 @@ type Address struct {
 	Type    string `json:"type"`
 }
 
-// Info contains information like what version the kubelet is running
+// Info contains node information like what version the kubelet is running
 type Info struct {
```

Comment:
The actual k8s node data type spec calls this `NodeInfo` (hence the change above). But gofmt complains if you start a method with the same word that is the type you're adding a method against (i.e., it doesn't want you to call something `node.NodeAction` and wants you to call it `node.Action` instead).

So, I kept the `Info` struct name here, but fixed the other definitions above so that they would actually correctly parse the node JSON output.